processor/snippet.go

```diff
@@ -172,28 +179,31 @@ func extractRelevantV3(res *fileJob, documentFrequencies map[string]int, relLeng
 
 		// Now we see if there are any nearby spaces to avoid us cutting in the
 		// middle of a word if we can avoid it
-		space, b := findNearbySpace(res, m.StartPos, 10)
-		if b {
-			m.StartPos = space
+		var b bool
+		m.StartPos, b = findSpaceRight(res, m.StartPos, SNIP_SIDE_MAX)
+		//m.StartPos = space
+		if !b {
+			fmt.Println("INFO - No space found on left. Cutting mid word ...")
 		}
-		space, b = findNearbySpace(res, m.EndPos, 10)
-		if b {
-			m.EndPos = space
+		m.EndPos, b = findSpaceLeft(res, m.EndPos, SNIP_SIDE_MAX)
+		//m.EndPos = space
+		if !b {
+			fmt.Println("INFO - No space found on right. Cutting mid word ...")
```

Comment:
This'll have to go.  How do you prefer to log this kinda stuff?
Esentially it doesn't matter now where `b` is true/false because the returned position will either be a space or the maximum possible in the snip.